gradle/project.gradle.kts

```diff
@@ -36,7 +37,7 @@ val kotlinVersion: String by extra {
             .resolvedConfiguration.firstLevelModuleDependencies
             .find { it.moduleName=="org.jetbrains.kotlin.jvm.gradle.plugin" }!!.moduleVersion
 }
-val javaSupportedVersions = arrayOf(JavaVersion.VERSION_1_9, JavaVersion.VERSION_1_10).also {
+val javaSupportedVersions = arrayOf(JavaVersion.VERSION_1_9, JavaVersion.VERSION_1_10, JavaVersion.VERSION_11).also {
```

Comment:
This needs to stop. **We** are in full control. The application is standalone with JDK packaged with it. There is absolutely no point to support more than 1 version.

Reply:
Done